### DISCUSSION TITLE
Welcome to AgentM Discussions!

### DISCUSSION THREAD

**Discussion by Stevenic** [Stevenic/agentm-js#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tuanvt** [comment]
Sound like exactly what micro agent should be, it should be a simple core interface that allow to create advanced agents if needed.

**Reply by Stevenic** [reply]
That's my thought as well... If you want to build a full agent framework you can. AgentM will have all of the parts but if you just want to add summarization to an existing application you can do that by importing 2 functions.  You pick and choose how much or how little of the library you want to use.
